spec/controllers/admin/admin_users_controller_spec.rb

```diff
@@ -539,6 +540,12 @@ def expect_changes_made_by(admin, log_items)
 
               subject.call
             end
+
+            it "does not send deleted comments to Akismet" do
+              expect(deleted_comment).to_not receive(:submit_spam)
```

Comment:
This passes even when the change is reverted. Something like this works better:
```suggestion
              comment.is_deleted = true
              comment.save!(validate: false)
              expect_any_instance_of(Comment).to_not receive(:submit_spam)
```